[2024-03-04T10:05:28.316+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data_to_S3 manual__2024-03-04T10:05:21.623738+00:00 [queued]>
[2024-03-04T10:05:28.331+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data_to_S3 manual__2024-03-04T10:05:21.623738+00:00 [queued]>
[2024-03-04T10:05:28.332+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-03-04T10:05:28.365+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_load_weather_data_to_S3> on 2024-03-04 10:05:21.623738+00:00
[2024-03-04T10:05:28.372+0000] {standard_task_runner.py:60} INFO - Started process 64 to run task
[2024-03-04T10:05:28.377+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'transform_load_weather_data_to_S3', 'manual__2024-03-04T10:05:21.623738+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/Weather_dag.py', '--cfg-path', '/tmp/tmppq_zqrjg']
[2024-03-04T10:05:28.380+0000] {standard_task_runner.py:88} INFO - Job 19: Subtask transform_load_weather_data_to_S3
[2024-03-04T10:05:28.474+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.transform_load_weather_data_to_S3 manual__2024-03-04T10:05:21.623738+00:00 [running]> on host 4f01ef8a3140
[2024-03-04T10:05:28.646+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stellar' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='transform_load_weather_data_to_S3' AIRFLOW_CTX_EXECUTION_DATE='2024-03-04T10:05:21.623738+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-04T10:05:21.623738+00:00'
[2024-03-04T10:05:29.273+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-04T10:05:29.286+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=transform_load_weather_data_to_S3, execution_date=20240304T100521, start_date=20240304T100528, end_date=20240304T100529
[2024-03-04T10:05:29.354+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-04T10:05:29.383+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
